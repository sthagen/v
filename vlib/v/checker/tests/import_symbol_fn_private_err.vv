import time { now, since }
fn main() {
  since(now())
}

fn method() {
    _ = map{'h':2}.exists_1('h')
}
