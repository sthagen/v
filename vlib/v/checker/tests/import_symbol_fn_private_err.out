vlib/v/checker/tests/import_symbol_fn_private_err.vv:1:20: error: module `time` function `since()` is private
    1 | import time { now, since }
      |                    ~~~~~
    2 | fn main() {
    3 |   since(now())
vlib/v/checker/tests/import_symbol_fn_private_err.vv:3:3: error: function `time.since` is private
    1 | import time { now, since }
    2 | fn main() {
    3 |   since(now())
      |   ~~~~~~~~~~~~
    4 | }
    5 |
vlib/v/checker/tests/import_symbol_fn_private_err.vv:7:20: error: method `map[string]int.exists_1` is private
    5 | 
    6 | fn method() {
    7 |     _ = map{'h':2}.exists_1('h')
      |                    ~~~~~~~~~~~~~
    8 | }
